src/main/java/org/demo/lesson3/tasks/abstract_classes/PerfectHoliday.java

```diff
@@ -0,0 +1,19 @@
+package org.demo.lesson3.tasks.abstract_classes;
+
+public abstract class PerfectHoliday {
+
+    private String city;
+
+    protected abstract void toDo();
+
+    public void setCity(String city) {
+        this.city = city;
+    }
+    public void getCity() {
```

Comment:
better to name this method provideCityName() as a method below, because by setter-getter pattern convention usually getCity() supposed to provide a value of city variable. Like:

public String getCity() {
   return city;
}